DefaultSettings/getDefaultSettings.m

```diff
@@ -65,22 +78,75 @@
     S.bounds.dist_trav.lower = [];
 end
 
-% upper bound on final time, in seconds
+% bound on final time, in seconds
+if ~isfield(S.bounds,'t_final')
+    S.bounds.t_final    = [];
+end
 if ~isfield(S.bounds.t_final,'upper')
-    S.bounds.t_final.upper = 2;
+    if strcmp(S.misc.gaitmotion_type,'HalfGaitCycle')
+        S.bounds.t_final.upper = 1;
+    else
+        S.bounds.t_final.upper = 2;
+    end
 end
-
-% lower bound on final time, in seconds
 if ~isfield(S.bounds.t_final,'lower')
     S.bounds.t_final.lower = 0.1;
 end
 
+if ~isfield(S.bounds,'activation_all_muscles')
+    S.bounds.activation_all_muscles = [];
+end
+
+% default coordinate bounds
+if ~isfield(S.bounds,'default_coordinate_bounds')
+    S.bounds.default_coordinate_bounds = 'Default_Coordinate_Bounds.csv';
+end
+
 % manually overwrite coordinate bounds
-if ~isfield(S.bounds,'coordinates')
-    S.bounds.coordinates = [];
+if ~isfield(S.bounds,'Qs')
+    S.bounds.Qs = [];
+end
+if ~isfield(S.bounds,'Qdots')
+    S.bounds.Qdots = [];
+end
+if ~isfield(S.bounds,'Qdotdots')
+    S.bounds.Qdotdots = [];
+end
+
+% minimal muscle activation, a number between 0 and 1
+if ~isfield(S.bounds.activation_all_muscles,'lower')
+    S.bounds.activation_all_muscles.lower = 0.05;
+end
+
+% activation of selected muscles
+if ~isfield(S.bounds,'activation_selected_muscles')
+    S.bounds.activation_selected_muscles = [];
+end
```

Comment:
I can see you can set the upper bound for activations using S.bounds.activation_selected_muscles, but would it be useful to be able to set the upper bounds of all muscles with just one setting (e.g. S.bounds.activation_all_muscles.upper)?

Reply:
I have added a setting for the upper bound